<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnalysisDlg</class>
 <widget class="QDialog" name="AnalysisDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>828</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>1000</x>
     <y>10</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/res/SoftIcon/Cancel.png</normaloff>:/res/SoftIcon/Cancel.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAll">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>450</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>全部</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClear">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>530</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>200</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>区域设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAreaSet">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>370</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>检测区域设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>610</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>70</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>通道</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="checkBox_Laser">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>250</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>激光使能</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>869</x>
     <y>132</y>
     <width>851</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>区域类型:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_AreaType">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>132</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>区域识别</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>区域屏蔽</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>温度识别区域</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Temperature">
   <property name="geometry">
    <rect>
     <x>917</x>
     <y>300</y>
     <width>83</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>37.3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Temp">
   <property name="geometry">
    <rect>
     <x>881</x>
     <y>300</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>温度:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
